### DISCUSSION TITLE
OBS does not like 240hz Monitors

### DISCUSSION THREAD

**Discussion by switchtonite** [obsproject/obs-studio#7074] [Development Questions] [answered]
Alright i recently purchased the new Samsung Neo g8 but when i stream live games obs seems to make all games jitter on my live stream. Is it possible to implement A Vulkan capture or make it Directx 12

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
